app/views/user/audits/_user_document.html.erb

```diff
@@ -1,16 +1,16 @@
 <% # Normalise auto-assigned variable %>
-<% audit ||= birth_records_user %>
+<% audit ||= user_document %>
 
 <div class="card audit" data-id="<%= audit.id %>">
   <div class="card-body">
     <h5 class="card-title">
       <% if audit.action == 'create'%>
-        <strong>Add</strong> birth record of <strong><%= audit.auditable.birth_record.full_name %></strong>
+        <strong>Add</strong> birth record of <strong><%= audit.auditable.document.full_name %></strong>
         <div class="text-muted">
            to your documents
         </div>
       <% else %>
-        <strong>Remove</strong> birth record of <strong><%= audit.auditable.birth_record.full_name %></strong>
+        <strong>Remove</strong> birth record of <strong><%= audit.auditable.document.full_name %></strong>
```

Comment:
This stuff will all need rejigging when another type of document comes up - no guarantee they'll have a `full_name`. But the audit comments are very birth record specific so it should be OK for the moment